src/manifest.ts

```diff
@@ -261,21 +261,30 @@ export class Manifest {
             )
         ).join(' ')
 
+        let commands = []
         switch (module.buildsystem) {
-            case undefined:
+            default:
             case 'autotools':
-                return this.getAutotoolsCommands(rebuild, buildArgs, configOpts)
+                commands = this.getAutotoolsCommands(rebuild, buildArgs, configOpts)
+                break
             case 'cmake':
             case 'cmake-ninja':
-                return this.getCmakeCommands(rebuild, buildArgs, configOpts)
+                commands = this.getCmakeCommands(rebuild, buildArgs, configOpts)
+                break
             case 'meson':
-                return this.getMesonCommands(rebuild, buildArgs, configOpts)
+                commands = this.getMesonCommands(rebuild, buildArgs, configOpts)
+                break
             case 'simple':
-                return this.getSimpleCommands(module['build-commands'], buildArgs)
+                commands = this.getSimpleCommands(module['build-commands'], buildArgs)
+                break
             case 'qmake':
                 throw new Error('Qmake is not implemented yet')
         }
-        throw new Error('Failed to build application')
+        /// Add the post-install commands if there are any
+        commands.push(
+            ... this.getSimpleCommands(this.module()['post-install'] || [], buildArgs)
```

Comment:
I would rename `getSimpleCommands` to something else

Reply:
at first it was used for simple build system, no suggestions for an alternative name though

Reply:
Something i noticed while doing this, is we have a ton of `new Command('flatpak', ['build', ...args])`, I was thinking of making that a subclass of Command but not a thing I plan to look at as part of this PR

Reply:
what do you think of `intoFlatpakBuildCommands`

Reply:
but then it could potentially be a helper function and not some Flatpak Manifest function, anyway, let us keep that change for later